src/main/target/SPEEDYBEEF4/target.c

```diff
@@ -28,7 +28,7 @@
 #include "drivers/timer_def.h"
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
-    DEF_TIM(TIM5, CH4, PC11,  TIM_USE_PPM,   0, 0), // PPM
+    DEF_TIM(TIM5, CH4, PA3,  TIM_USE_PPM,   0, 0), // PPM
```

Comment:
What is the reason for this change? If this is a second version of the hardware, then adding it as a variant should be considered.

Reply:
Was the first version ever available to customers?

Reply:
yes

Reply:
Ok, this should be changed to define a new target variant then, so that owners of the first version of the board aren't left out.